.hound.yml

```diff
@@ -0,0 +1,5 @@
+javascript:
+  enabled: false
```

Comment:
@ricahcyuzuzo why did you disable JavaScript here?

Reply:
@karamuka For enablling ESLint styling guides only not javascript styling guides